std.ddoc

```diff
@@ -37,13 +44,14 @@ function listanchors()
     values.sort();
 
     var newText = "";
-    for(var i = 0; i < values.length; i++) {
+    for (var i in values) {
```

Comment:
Hmm, so it's slower, and iterates in order of element creation rather than index order.